app/views/internal/configs/show.html.erb

```diff
@@ -26,193 +26,23 @@
         <div class="card mt-3">
           <%= render partial: "card_header",
                      locals: {
-                       header: "Community Content",
```

Comment:
Alphabetized the site config page so that it's easier to find things. Combine and rename sections where applicable.
i.e. added twitter_hashtag to the Social Media section and renamed Mailchimp section to Newsletter